web/src/click_handlers.js

```diff
@@ -415,7 +412,7 @@ export function initialize() {
     // Mute topic in a muted stream
     $("body").on("click", ".message_header .stream_muted.on_hover_topic_mute", (e) => {
         e.stopPropagation();
-        muted_topics_ui.mute_or_unmute_topic(
+        user_topics_ui.mute_or_unmute_topic(
             $(e.target),
```

Comment:
`mute_or_unmute_topic` is related to `message_header` + `recent_topics_table`. Should we rename this here since we will replace the current UI with the proposed tabs UI? We're finally going to remove this function altogether (once the feature is live)

proposed UI: https://chat.zulip.org/#narrow/stream/101-design/topic/Follow.20Topics.20UI/near/1599171

---

After discussion in CZO (DM) with @abhijeetbodas2001 ; added a comment in the file:
```javascript
// This function is only called from the message header & recent_topics table
// and will be removed once we migrate to the new UI for selecting visibility policy
// for a topic.
export function mute_or_unmute_topic($elt, visibility_policy) {
...
}